### DISCUSSION TITLE
카테고리 기능

### DISCUSSION THREAD

**Discussion by daehwan2** [code-etc/Documents#8] [Ideas]
카테고리로 어떤 고정 카테고리를 넣으면 좋을지 말씀해주세요~😀
@daehwan2 @tjwjdgks @on-Sync @yunjin-kim

**Comment by daehwan2** [comment]
- 음식
- 남자 연예인
- 여자 연예인
- 만화
- 영화
- 드라마
- 자동차

**Comment by tjwjdgks** [comment]
* 게임
* 능력(ex 초능력, 뛰어난 능력등)
